tests/py/dynamo/conversion/test_chunk_aten.py

```diff
@@ -75,8 +78,102 @@ def forward(self, input):
         self.run_test(
             TestChunk(),
             input,
+            use_dynamo_tracer=True,
         )
 
+#######################Dynamic cases################
+
+    # @parameterized.expand(
+    #     [
+    #         ((1,), (1,), (3,), 3, 0),
+    #         ((3,), (3,), (4,), 3, 0),
+    #         ((4,), (4,), (6,), 3, 0),
+    #         ((6,), (6,), (9,), 3, 0),
+    #         ((3,), (3,), (4,), 1, -1),
+    #         ((3,), (3,), (4,), 3, -1),
+    #         ((3,), (3,), (4,), 4, -1),
+    #     ]
+    # )
+    # def test_chunk_1D(self, min_shape, opt_shape, max_shape, chunks, dim):
```

Comment:
Probably it's better to add a unittest.skip similar to https://github.com/pytorch/TensorRT/blob/95cc532aa6a11c8bb55bc43bc2b04a76d264577e/tests/py/dynamo/conversion/test_evaluators.py#L13 and add a link to the issue you created.